src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  box-sizing: border-box;
+  max-width: $cell-size * $col-count + $gap * ($col-count - 1) + $padding * 2;
+  margin: auto;
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    transition: 0.5s;
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day})::before {
+        content: "#{$day}";
+      }
+    }
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: $background-color-hover;
+      cursor: pointer;
+    }
+  }
+}
+
+$week-days: "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+@each $day,
+  $number in $week-days {
+  .calendar--start-day--#{$day} {
```

Comment:
Here you use full name on your selector, but above you use '&' to refer to parent selector. Make sense to transfer these code into `.calendar` styles section and use here '&' too.